### DISCUSSION TITLE
IntegrationApp not installing properly on iOS Device

### DISCUSSION THREAD

**Discussion by murigregori** [appium/appium#19792] [Q&A]
Hi! Needing help on this for my iOS automation. Trying to update the WebDriverAgent xcode project under `appium-webdriveragent` folder and when I try to build the IntegrationApp, i'm not able to install it on my device. It always shows this error: 

<img width="869" alt="MicrosoftTeams-image (108)" src="https://github.com/appium/appium/assets/20291195/c85a6123-22aa-4c21-b45c-1424bbac45ea">

Im using Xcode 15.1 and iOS device is iPhone XR (iOS 17.0.3). There's no certificate showing up in my VPN/Device Management either on my device so I cant trust the certificate from xcode. Asking here hoping that someone else has encountered it and has a possible fix. Thank yoU!

**Comment by KazuCocoa** [comment]
I haven't seen such a showing-up case. The IntegrationApp itself is a regular iOS app, so possibly it is Xcode's signature error handling-related issue for the message dialog itself. Btw, if you re-install/uninstall the app, existing trust certificate section goes away from the device management section on iOS.
